images/ext-auth-example-authsvc/Makefile

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 The Kubernetes Authors. All rights reserved.
```

Comment:
2022 happy new year 🥳🥳 (I keep adding 2021 on files as well...)

Reply:
Happy new year @rikatz 🥳 . Updated the file with 2022.